Failing dashboard Cypress test

First mentioned by @tino097 in https://github.com/ckan/ckan/pull/6557#issuecomment-1044680100, this has started to pop up again ([example](https://github.com/ckan/ckan/runs/5751402784?check_suite_focus=true))


```
Running:  modules/dashboard.spec.js                                                     (10 of 24)
[3711:0330/092112.757393:ERROR:bus.cc(392)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")


  ckan.modules.DashboardModule()
2022-03-30 09:21:16,159 INFO  [ckan.config.middleware.flask_app]  200 / render time 0.158 seconds
2022-03-30 09:21:16,161 INFO  [werkzeug] 127.0.0.1 - - [30/Mar/2022 09:21:16] "GET / HTTP/1.1" 200 -
2022-03-30 09:21:16,300 INFO  [ckan.config.middleware.flask_app] 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
2022-03-30 09:21:16,404 INFO  [ckan.config.middleware.flask_app]  404 /TEST_TEMPLATE_HEAD_END.css render time 0.112 seconds
2022-03-30 09:21:16,406 INFO  [werkzeug] 127.0.0.1 - - [30/Mar/2022 09:21:16] "GET /TEST_TEMPLATE_HEAD_END.css HTTP/1.1" 404 -
2022-03-30 09:21:16,425 INFO  [ckan.config.middleware.flask_app]  200 /webassets/base/293bac58_main.css render time 0.017 seconds
2022-03-30 09:21:16,434 INFO  [werkzeug] 127.0.0.1 - - [30/Mar/2022 09:21:16] "GET /webassets/base/293bac58_main.css HTTP/1.1" 200 -
2022-03-30 09:21:16,451 INFO  [ckan.config.middleware.flask_app]  200 /webassets/vendor/f3b8236b_select2.css render time 0.007 seconds
2022-03-30 09:21:16,454 INFO  [werkzeug] 127.0.0.1 - - [30/Mar/2022 09:21:16] "GET /webassets/vendor/f3b8236b_select2.css HTTP/1.1" 200 -
2022-03-30 09:21:16,470 INFO  [ckan.config.middleware.flask_app]  200 /webassets/vendor/0b01aef1_font-awesome.css render time 0.010 seconds
2022-03-30 09:21:16,474 INFO  [werkzeug] 127.0.0.1 - - [30/Mar/2022 09:21:16] "GET /webassets/vendor/0b01aef1_font-awesome.css HTTP/1.1" 200 -
2022-03-30 09:21:16,495 INFO  [ckan.config.middleware.flask_app]  200 /webassets/vendor/8c3c143a_jquery.js render time 0.017 seconds
2022-03-30 09:21:16,496 INFO  [werkzeug] 127.0.0.1 - - [30/Mar/2022 09:21:16] "GET /webassets/vendor/8c3c143a_jquery.js HTTP/1.1" 200 -
2022-03-30 09:21:16,509 INFO  [ckan.config.middleware.flask_app]  200 /webassets/vendor/6137ee05_vendor.js render time 0.006 seconds
2022-03-30 09:21:16,514 INFO  [werkzeug] 127.0.0.1 - - [30/Mar/2022 09:21:16] "GET /webassets/vendor/6137ee05_vendor.js HTTP/1.1" 200 -
2022-03-30 09:21:16,530 INFO  [ckan.config.middleware.flask_app]  200 /webassets/vendor/f0bcb805_bootstrap.js render time 0.013 seconds
2022-03-30 09:21:16,531 INFO  [werkzeug] 127.0.0.1 - - [30/Mar/2022 09:21:16] "GET /webassets/vendor/f0bcb805_bootstrap.js HTTP/1.1" 200 -
2022-03-30 09:21:16,541 INFO  [ckan.config.middleware.flask_app]  200 /webassets/base/d2e3f2b0_main.js render time 0.007 seconds
2022-03-30 09:21:16,541 INFO  [werkzeug] 127.0.0.1 - - [30/Mar/2022 09:21:16] "GET /webassets/base/d2e3f2b0_main.js HTTP/1.1" 200 -
2022-03-30 09:21:16,763 INFO  [ckan.config.middleware.flask_app]  200 /webassets/base/601a4ef8_ckan.js render time 0.011 seconds
2022-03-30 09:21:16,763 INFO  [werkzeug] 127.0.0.1 - - [30/Mar/2022 09:21:16] "GET /webassets/base/601a4ef8_ckan.js HTTP/1.1" 200 -
2022-03-30 09:21:16,792 INFO  [ckan.config.middleware.flask_app]  200 /webassets/base/cd2c0272_tracking.js render time 0.012 seconds
2022-03-30 09:21:16,793 INFO  [werkzeug] 127.0.0.1 - - [30/Mar/2022 09:21:16] "GET /webassets/base/cd2c0272_tracking.js HTTP/1.1" 200 -
2022-03-30 09:21:16,927 INFO  [ckan.config.middleware.flask_app]  200 /base/vendor/font-awesome/fonts/fontawesome-webfont.woff2 render time 0.012 seconds
2022-03-30 09:21:16,927 INFO  [werkzeug] 127.0.0.1 - - [30/Mar/2022 09:21:16] "GET /base/vendor/font-awesome/fonts/fontawesome-webfont.woff2?v=4.7.0 HTTP/1.1" 200 -
2022-03-30 09:21:16,956 INFO  [ckan.config.middleware.flask_app]  200 /base/images/bg.png render time 0.027 seconds
2022-03-30 09:21:16,956 INFO  [werkzeug] 127.0.0.1 - - [30/Mar/2022 09:21:16] "GET /base/images/bg.png HTTP/1.1" 200 -
2022-03-30 09:21:16,972 INFO  [ckan.config.middleware.flask_app] 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
2022-03-30 09:21:17,107 INFO  [ckan.config.middleware.flask_app]  404 /images/ckan_logo_fullname_long.png render time 0.149 seconds
2022-03-30 09:21:17,111 INFO  [werkzeug] 127.0.0.1 - - [30/Mar/2022 09:21:17] "GET /images/ckan_logo_fullname_long.png HTTP/1.1" 404 -
2022-03-30 09:21:17,130 INFO  [ckan.config.middleware.flask_app]  200 /base/images/od_80x15_blue.png render time 0.017 seconds
2022-03-30 09:21:17,137 INFO  [werkzeug] 127.0.0.1 - - [30/Mar/2022 09:21:17] "GET /base/images/od_80x15_blue.png HTTP/1.1" 200 -
2022-03-30 09:21:17,156 INFO  [ckan.config.middleware.flask_app]  200 /base/images/placeholder-420x220.png render time 0.017 seconds
2022-03-30 09:21:17,157 INFO  [werkzeug] 127.0.0.1 - - [30/Mar/2022 09:21:17] "GET /base/images/placeholder-420x220.png HTTP/1.1" 200 -
2022-03-30 09:21:17,171 INFO  [ckan.config.middleware.flask_app]  200 /base/images/background-tile.png render time 0.013 seconds
2022-03-30 09:21:17,172 INFO  [werkzeug] 127.0.0.1 - - [30/Mar/2022 09:21:17] "GET /base/images/background-tile.png HTTP/1.1" 200 -
2022-03-30 09:21:17,196 INFO  [ckan.config.middleware.flask_app]  200 /base/images/ckan-logo-footer.png render time 0.022 seconds
2022-03-30 09:21:17,196 INFO  [werkzeug] 127.0.0.1 - - [30/Mar/2022 09:21:17] "GET /base/images/ckan-logo-footer.png HTTP/1.1" 200 -
2022-03-30 09:21:17,501 INFO  [ckan.config.middleware.flask_app]  200 /api/1/util/snippet/dashboard.html render time 0.084 seconds
2022-03-30 09:21:17,507 INFO  [werkzeug] 127.0.0.1 - - [30/Mar/2022 09:21:17] "GET /api/1/util/snippet/dashboard.html HTTP/1.1" 200 -
    .initialize()
      ✓ should bind callback methods to the module (1911ms)
    .show()
      ✓ should append the popover to the document body (71ms)
    .search
      1) should filter based on query


  2 passing (7s)
  1 failing

  1) ckan.modules.DashboardModule()
       .search
         should filter based on query:
     AssertionError: Timed out retrying after 4000ms: Expected to find element: `#fixture #followee-filter .input-group input.inited`, but never found it.
      at Context.eval (http://localhost:5000/__cypress/tests?p=cypress/integration/modules/dashboard.spec.js:148:10)




  (Results)

  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ Tests:        3                                                                                │
  │ Passing:      2                                                                                │
  │ Failing:      1                                                                                │
  │ Pending:      0                                                                                │
  │ Skipped:      0                                                                                │
  │ Screenshots:  1                                                                                │
  │ Video:        true                                                                             │
  │ Duration:     6 seconds                                                                        │
  │ Spec Ran:     modules/dashboard.spec.js                                                        │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘


  (Screenshots)

  -  /home/runner/work/ckan/ckan/cypress/screenshots/modules/dashboard.spec.js/ckan.m     (1280x720)
     odules.DashboardModule() -- .search -- should filter based on query (failed).png               

```

--- a/ckan/public/base/javascript/modules/dashboard.js
+++ b/ckan/public/base/javascript/modules/dashboard.js
@@ -23,10 +23,11 @@ this.ckan.module('dashboard', function ($) {
         on('click', this._onShowFolloweeDropdown);
       var title = this.button.prop('title');
 
-      this.button.popover({
+      this.button.popover = new bootstrap.Popover(document.querySelector('#followee-filter .btn'), {
           placement: 'bottom',
           html: true,
           template: '<div class="popover" role="tooltip"><div class="popover-arrow"></div><h3 class="popover-header"></h3><div class="popover-body followee-container"></div></div>',
+          customClass: 'popover-followee',
           sanitizeFn: function (content) {
             return DOMPurify.sanitize(content, { ALLOWED_TAGS: [
               "form", "div", "input", "footer", "header", "h1", "h2", "h3", "h4",
@@ -80,6 +81,7 @@ this.ckan.module('dashboard', function ($) {
      * Returns nothing.
      */
     _onSearchKeyUpTimeout: function() {
+      this.popover = this.button.popover.tip;
       var input = $('input', this.popover);
       var q = input.val().toLowerCase();
       if (q) {

